api/src/main/java/org/pmiops/workbench/db/model/DbInstitutionEmailAddress.java

```diff
@@ -0,0 +1,49 @@
+package org.pmiops.workbench.db.model;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "institution_email_address")
+public class DbInstitutionEmailAddress {
+
+  @Id
```

Comment:
I like how this gives you the flexibility to have an email belong to more than one institution someday. (This should be in your test planning too).

Reply:
tests now show this